SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -921,3 +921,36 @@ void CFlowOutput::Set_CpInverseDesign(CSolver *solver, CGeometry *geometry, CCon
   SetHistoryOutputValue("CP_DIFF", PressDiff);
 
 }
+
+su2double CFlowOutput::GetQ_Criterion(su2double** VelocityGradient) const {
```

Comment:
I think the Q criterion was broken, j-i was being used before it was set.
I copy pasted the changes from @economon on #704 and I moved the method to the common parent class CFlowOutput

Reply:
Thanks! That was already on my agenda ..